tests/generate-certificates.sh

```diff
@@ -54,17 +54,11 @@ GROUP_SIZE=4
 #overrides default is set
 parse-args "${@}"
 
-echo "*************************************************"
-echo "         generating dh param                     "
-echo "*************************************************"
-#using low values like 128 here and below as this is for unit tests and our goal to save running time. For real applications 2048 recommended
-openssl dhparam -out dh.pem 128
-
```

Comment:
explicit dh params key exchange is less performant than elliptic curve. Google Chrome removed this type of key exchange in 2016 so I don't think we should support it.